types/types.go

```diff
@@ -47,9 +47,12 @@ type ImageDestination interface {
 // Image is a Docker image in a repository.
 type Image interface {
 	// ref to repository?
+	// GetManifest is like ImageSource.GetManifest, but the result is cached; it is OK to call this however often you need.
+	GetManifest() ([]byte, error)
+	// GetSignatures is like ImageSource.GetSignatures, but the result is cached; it is OK to call this however often you need.
+	GetSignatures() ([][]byte, error)
```

Comment:
I'm fine merging this as-is given we could remove at least `Get` (https://github.com/projectatomic/skopeo/pull/57#discussion_r63395075)